web/elm/src/Application/Application.elm

```diff
@@ -510,3 +542,48 @@ routeMatchesModel route model =
 
         _ ->
             False
+
+
+wallLinks : String -> List (Html.Html msg)
+wallLinks msg =
+    let
+        tokens = String.split " " msg
+
+        stripTrailingPunct original =
+            let
+                punctuations = [ '.', ',' ]
+
+                dropWhileEnd s =
+                    case String.uncons (String.reverse s) of
+                        Just ( c, restRev ) ->
+                            if List.member c punctuations then
+                                let
+                                    trimmed = String.reverse restRev
+                                    ( url, trailing ) = dropWhileEnd trimmed
+                                in
+                                ( url, String.fromChar c ++ trailing )
+                            else
+                                ( s, "" )
+
+                        Nothing ->
+                            ( s, "" )
+            in
+            dropWhileEnd original
+
+        render t =
+            if String.startsWith "http://" t || String.startsWith "https://" t then
+                let
+                    ( url, trailing ) = stripTrailingPunct t
+                    anchor = Html.a [ href url, target "_blank", rel "noopener noreferrer" ] [ Html.text url ]
+                in
+                if trailing == "" then
+                    [ anchor ]
+                else
+                    [ anchor, Html.text trailing ]
+            else
+                [ Html.text t ]
```

Comment:
Whitespace around the link is being stripped: 
<img width="298" height="63" alt="image" src="https://github.com/user-attachments/assets/717275f2-0b8f-4e76-814c-3ef664ddaaef" />

Setting the message:
```
fly -t l set-wall -m "visit https://example.com for details" --ttl 1h
```